hoot-core-test/src/test/cpp/hoot/core/algorithms/LongestCommonNodeStringTest.cpp

```diff
@@ -60,13 +60,19 @@ class LongestCommonNodeStringTest : public HootTestFixture
 
 public:
 
+  LongestCommonNodeStringTest()
+    : HootTestFixture("test-files/algorithms/",
+                      UNUSED_PATH)
+  {
+  }
+
```

Comment:
This is one of the biggest changes, passing the input path and output path to the base class constructor.  Just about every test has one of these.